aiozk/recipes/base_lock.py

```diff
@@ -15,7 +15,18 @@ class BaseLock(SequentialRecipe):
         if timeout is not None:
             time_limit = time.time() + timeout
 
-        await self.create_unique_znode(znode_label)
+        try:
+            await self.create_unique_znode(znode_label)
+        except Exception as ex:
+            log.debug('unknown exception: %s', ex)
+            while True:
+                try:
+                    await self.delete_garbage_znodes(znode_label)
+                except Exception as e:
+                    log.debug('unknown exception: %s', e)
+                else:
+                    break
```

Comment:
This `break` can be safely moved inside try block

Reply:
Ok that is more concise.